### ISSUE TITLE
Intermittent issues with "subscriptions"

### ISSUE THREAD

**Issue by asavah** [jdf76/plugin.video.youtube#818]
To reproduce:
Navigate to "Subscriptions" and try to enter (_not_ "my subscriptions")

![screenshot000](https://user-images.githubusercontent.com/5447220/81241621-6fa3b700-9013-11ea-9274-7a58319bce6d.png)

After trying like 3 times I can actually proceed. It's intermittent, sometimes it works on the first try, sometimes I have to try a few times.

```
2020-05-07 03:24:42.461 T:1283    DEBUG <general>: [plugin.video.youtube] Selecting YouTube config "All"
2020-05-07 03:24:42.462 T:1283    DEBUG <general>: [plugin.video.youtube] Access token count: |2| Refresh token count: |2|
2020-05-07 03:24:42.462 T:1283    DEBUG <general>: [plugin.video.youtube] User is logged in
2020-05-07 03:24:42.462 T:1283    DEBUG <general>: [plugin.video.youtube] [data] v3 request: |GET| path: |subscriptions| params: |{'part': 'snippet', 'maxResults': '50', 'order': 'alphabetical', 'mine': 'true'}| post_data: |None|
2020-05-07 03:24:42.546 T:1213    DEBUG <general>: ------ Window Init (DialogBusy.xml) ------
2020-05-07 03:24:43.356 T:1283    ERROR <general>: [plugin.video.youtube] Unknown kind 'youtube#SubscriptionListResponse'
```

Debug log:
Here I try 3 times, the third try's the charm )
https://paste.kodi.tv/fibacenewi.kodi

**Comment by asavah** [comment]
After a quick grep of the code I think it's some mishap on youtube's side or some subtle api change where their backends randomly return  youtube#`S`ubscriptionListResponse (note the capital S) instead of youtube#subscriptionListResponse

This quick and dirty patch seems to solve the issue:
```
--- v3.py.orig  2020-05-07 03:44:38.834540658 +0300
+++ v3.py       2020-05-07 03:45:10.258167331 +0300
@@ -279,7 +279,7 @@

     kind = json_data.get('kind', '')
     if kind == u'youtube#searchListResponse' or kind == u'youtube#playlistItemListResponse' or \
-            kind == u'youtube#playlistListResponse' or kind == u'youtube#subscriptionListResponse' or \
+            kind == u'youtube#playlistListResponse' or kind == u'youtube#subscriptionListResponse' or kind == u'youtube#SubscriptionListResponse' or \
             kind == u'youtube#guideCategoryListResponse' or kind == u'youtube#channelListResponse' or \
             kind == u'youtube#videoListResponse' or kind == u'youtube#activityListResponse' or \
             kind == u'youtube#commentThreadListResponse' or kind == u'youtube#commentListResponse':
```

**Comment by anxdpanic** [comment]
Thanks for catching that, so easy to overlook when it's something like that.
Should be fixed by #820

**Comment by asavah** [comment]
Thanks for your awesome work.
